scripts/npcs/village/priestesslinota.lua

```diff
@@ -150,15 +150,15 @@ local function priestess_talk(npc, ch)
                     ch:inv_change("Unholy Crystals", -1)
                     say("Let me see...")
                     say("This is... very alarming. The order needs to be "
-                        .. "informed about that. I'll write a letter "
+                        .. "informed about this. I'll write a letter "
                         .. "explaining what we found out.")
                     say("Once the path above the mountains is open again, "
                         .. "please bring this letter to the shrine of "
                         .. "Mountains Watch north of here.")
                         -- LATER: remove the part about closed path after we
                         -- added more content
                     say("Your journey might be dangerous, so I'll grant "
-                        .. "you further knowledge in the gods powers.")
+                        .. "you further knowledge in the god's powers.")
```

Comment:
Since there are multiple gods, this should stay plural.

Reply:
What would you suggest changing here? It's confusing because it looks like it should be possessive and plural.

Reply:
I just found out that a word can be shown as possessive and plural at the same time if you put the apostrophe at the end of the word. I am going to change the sentence to: "Your journey might be dangerous, so I'll grant you further knowledge of the gods' powers.".

Reply:
Yes. Alternatively you could write "of the powers of the gods", but I prefer your solution.